File: fibonacci.rb
def fibonacci(n)
  if n <= 1
    n
  else
    fibonacci(n - 1) + fibonacci(n - 2)
  end
end

def main
  puts fibonacci(10)
  puts fibonacci(20)
  puts fibonacci(30)
  puts fibonacci(35)
  puts fibonacci(40)  
end

main

File: fibonacci.js
function fibonacci(n) {
  if (n <= 1) {
    return n
  } else {
    return fibonacci(n - 1) + fibonacci(n - 2)
  }
}

function main() {
  console.log(fibonacci(10))
  console.log(fibonacci(20))
  console.log(fibonacci(30))
  console.log(fibonacci(35))
  console.log(fibonacci(40))
}

main()
